pygeoapi/api.py

```diff
@@ -125,6 +125,10 @@ def root(self, headers, args):
         ]
 
         if format_ == 'html':  # render
+            for link in fcm['links']:
+                if 'json' in link['type']:
+                    link['href'] += '?f=json'
```

Comment:
is this better written as `link['href'] = ''.join((link['href'], '?f=json'))`?  I'm not sure what the current verdict is around 'adding' strings in Python.

Reply:
From what I read both `+` and `join` can be used, with `join` being marginally faster with many strings to concatenate. For readablity `+` is preferred.